modules/custom/az_person/css/az_person.css

```diff
@@ -27,7 +27,7 @@
 	display: none;
 }
 
-.az-person-grid .node--type-az-person.node--view-mode-az-row .az-person-row-img .contextual button {
+.az-person-row .node--type-az-person.node--view-mode-az-row .az-person-row-img .contextual button {
```

Comment:
Could we remove `az-person-row-img` from the group classes and replace it with `az-person-image-link` here?

If we wanted to, we could even consolidate this rule with the one above and target `az-person-image-link` since we are only using it on these two displays. A selector such as `.contextual-region.node--type-az-person .az-person-image-link .contextual button` would work.

Reply:
I think I would prefer a `.hide-contextual-links` class